### PR TITLE
Rename HTML files in src/test/resources/pages that start with upper case #5103 

### LINKED ISSUE #5103: Rename HTML files in src/test/resources/pages that start with upper case
The files in `src/test/resources/pages` should all start with a lower case. There are some `Instructor*.html` that don't follow it.

### PR DESCRIPTION
**Pull request by VIthulan**
Fixes #5103

### CODE REVIEW

**Comment by junhaoyap** [comment]
@VIthulan Please check the process - the PR title is a little off and the description is slightly off too, fix it and ping me and I will review

**Comment by VIthulan** [comment]
@junhaoyap fixed PR title and description

**Comment by junhaoyap** [comment]
@VIthulan awesome - I'll set it to review :)

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Important: When a version is specified, please install that version instead of t
 ##Setting up the dev server
 `Dev server` means running the server in your own computer.
 
-1. Fork our repo at `https://github.com/TEAMMATES/repo. Clone that fork to your hard disk.
+1. Fork our repo at https://github.com/TEAMMATES/repo. Clone that fork to your hard disk.
```
Comment (junhaoyap) [inline devdocs/settingUp.md]: Please try not to fix things unrelated to the issue that you're working on - it makes it harder if we ever need to revert a PR - could you revert this commit and check if an issue is already opened regarding this and if not please open one instead

**Comment by junhaoyap** [comment]
@VIthulan just a minor comment - go ahead and fix it and we should be good to go to set this to merge

**Comment by wkurniawan07** [comment]
@VIthulan two issues:
1. There are 3 more files that you forgot to rename.
2. You would've to modify tests to reflect these renames, but I guess @junhaoyap can do it for you.

**Comment by junhaoyap** [comment]
@VIthulan @wkurniawan07 yes I probably will do it for him - that might be too much for a first timer task

By the way you're right there are 3 more files, I was only looking at those with ```I``` so I missed the others, thanks for the great catch :100:

**Comment by junhaoyap** [comment]
Reminder, I will be closing this if not done by V5.75 @VIthulan

**Comment by wkurniawan07** [comment]
Closing due to inactivity.

**Pull request closed without merging**